### DISCUSSION TITLE
I am Owner on my Server but cant edit groups help pls

### DISCUSSION THREAD

**Discussion by Raikss** [Pryaxis/TShock#1794] [Q&A] [answered]
Hey, I am Owner on my new Trraria server but I can't switch the groups from users or my own.

It is telling me I don´t have access to this command.

I did the /setup thing at first but cant switch around groups pls help :(

**Comment by hakusaro** [comment]
Are you logged in with the same user you set the server up with?

**Reply by Raikss** [reply]
Yes i am

**Reply by Raikss** [reply]
i am in server group owner with that account normal commands like give and time are working fine just the group not

**Reply by hakusaro** [reply]
Try `/su` or `/sudo` and then your command. Either will boost you to superadmin temporarily.

**Reply by Raikss** [reply]
yea i get syntax error with /sudo /user add "name" Admin

**Comment by hakusaro** [comment, marked as answer]
Try /su instead to temporarily switch to a superuser.

Lucas Nicodemus
http://keybase.io/xn

**Reply by Raikss** [reply]
ok that fixed it thanks :D
